clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntity.java

```diff
@@ -202,6 +203,8 @@ public void releaseCache(String name) throws ClusteredTierReleaseException, Time
   public void destroyCache(String name) throws ClusteredTierDestructionException, TimeoutException {
     try {
       invokeInternal(timeouts.getLifecycleOperationTimeout(), messageFactory.destroyServerStore(name), true);
+    } catch (ResourceBusyException e) {
```

Comment:
Idea here is that this is a _valid_ outcome, not the result of messed up state server side. If we agree that catching these differently makes sense, I would probably revise a number of other locations in that same class.